plugins/kubernetes/api-report.md

```diff
@@ -259,6 +261,12 @@ export interface KubernetesApi {
   getWorkloadsByEntity(
     request: WorkloadsByEntityRequest,
   ): Promise<ObjectsByEntityResponse>;
+  // (undocumented)
+  proxy(options: {
+    clusterName: string;
+    path: string;
+    init?: RequestInit;
```

Comment:
What's the `RequestInit` for?

Reply:
If you want to use a different HTTP verb, send a request body, add headers, etc. Same reason you'd want it when making a normal `fetch` call. This is discussed in #16065